private/model/api/load.go

```diff
@@ -42,8 +42,10 @@ func loadAPI(modelPath, baseImport string) (*API, error) {
 		BaseCrosslinkURL: "https://docs.aws.amazon.com",
 	}
 
-	err := attachModelFiles(modelPath,
-		modelLoader{"api-2.json", a.Attach, true},
+	modelFile := filepath.Base(modelPath)
+	modelDir := filepath.Dir(modelPath)
+	err := attachModelFiles(modelDir,
+		modelLoader{modelFile, a.Attach, true},
 		modelLoader{"docs-2.json", a.AttachDocs, true},
```

Comment:
If we allow for custom `api-2.json` why not the others?

Reply:
Currently there is no other way of specifying the alternative filenames. Could consider that as a future feature.